UPGRADE-2.x.md

```diff
@@ -18,6 +18,8 @@ UPGRADE 2.x
 | `Sonata\CacheBundle\Adapter\MongoCache` | `Sonata\Cache\Adapter\Cache\MongoCache` |
 | `Sonata\CacheBundle\Adapter\NoopCache` | `Sonata\Cache\Adapter\Cache\NoopCache` |
 | `Sonata\CacheBundle\Adapter\PRedisCache` | `Sonata\Cache\Adapter\Cache\PRedisCache` |
+| `Sonata\CacheBundle\Twig\TwigTemplate13` | `none` |
```

Comment:
If the hole class is deprecated you should only add the `@deprecated` to the class phpdoc.

You can put the following message: `__CLASS__.' is deprecated since 2.x and will be removed in 3.0.'`